storybook/pages/MembersDropdownPage.qml

```diff
@@ -0,0 +1,304 @@
+
+                            expression: {
+                                membersDropdown.selectedKeys
+                                membersDropdown.searchText
+
+                                if (membersDropdown.selectedKeys.indexOf(model.pubKey) > -1)
+                                    return true
+
+                                const filter = membersDropdown.searchText.toLowerCase()
+                                return matchesAlias(model.alias.toLowerCase(), filter)
+                                         || model.displayName.toLowerCase().includes(filter)
+                                         || model.ensName.toLowerCase().includes(filter)
+                                         || model.localNickname.toLowerCase().includes(filter)
+                                         || model.pubKey.toLowerCase().includes(filter)
+                            }
+                        }
+                    ]
+                }
+
+                Component.onCompleted: open()
+            }
+        }
+    }
+
+    LogsAndControlsPanel {
+        SplitView.minimumHeight: 100
+        SplitView.preferredHeight: 250
+
+        Loader {
+            active: loader.item
+
+            anchors.left: parent.left
+            anchors.right: parent.right
+
+            sourceComponent: ColumnLayout {
+                readonly property MembersDropdown membersDropdown: loader.item
+
+                RowLayout {
+                    Label {
+                        text: "maximum list height:"
+                    }
+
+                    Slider {
+                        id: maxListHeightSlider
+                        from: 100
+                        to: 500
+                        stepSize: 1
+
+                        Component.onCompleted: {
+                            value = membersDropdown.maximumListHeight
+                            membersDropdown.maximumListHeight
+                                    = Qt.binding(() => value)
+                        }
+                    }
+
+                    Label {
+                        text: maxListHeightSlider.value
+                    }
+                }
+
+                RowLayout {
+                    Label {
+                        text: "margins:"
+                    }
+
+                    Slider {
+                        id: marginsSlider
+                        from: -1
+                        to: 50
+                        stepSize: 1
+
+                        Component.onCompleted: {
+                            value = membersDropdown.margins
+                            membersDropdown.margins = Qt.binding(() => value)
+                        }
+                    }
+
+                    Label {
+                        text: marginsSlider.value
+                    }
+                }
+
+                RowLayout {
+                    Label {
+                        text: "bottom inset:"
+                    }
+
+                    Slider {
```

Comment:
When I modify that, there is this small issue with the button:

![Screenshot 2023-04-18 at 13 56 22](https://user-images.githubusercontent.com/97019400/232774385-9f4f184a-cd49-4635-b199-5e0c68385ada.png)

Reply:
yeah, but this slider is only to observe the behavior, normally it's just set to desired value